### PR TITLE
Add inventory file for openvswitch 

### PR DESCRIPTION
**Pull request by NilashishC**
Depends-On: https://github.com/ansible/network-ee/pull/125

### CODE REVIEW

File: tests/integration/inventories/openvswitch.yaml [hunk 1]
```diff
@@ -0,0 +1,18 @@
...
+    localhost:
+      ansible_connection: local
+      ansible_python_interpreter: /bin/python3.8
+appliance:
+  hosts:
+    openvswitch:
+      ansible_host: openvswitch.example.org
+      ansible_user: zuul
+  vars:
+    ansible_network_os: openvswitch
```
Comment (pabelanger) [inline tests/integration/inventories/openvswitch.yaml:13]: we should use fqcn: openvswitch.openvswitch.openvswitch

File: tests/integration/inventories/openvswitch.yaml [hunk 2]
```diff
@@ -0,0 +1,18 @@
...
+appliance:
+  hosts:
+    openvswitch:
+      ansible_host: openvswitch.example.org
+      ansible_user: zuul
+  vars:
+    ansible_network_os: openvswitch.openvswitch.openvswitch
+    # NOTE(pabelanger): With stable-2.9 / stable-2.11 we need to force
+    # /bin/python3.8 and not discover /usr/libexec/platform-python (python3.6)
+    # ansible_python_interpreter: /bin/python3.8
```
Comment (pabelanger) [inline tests/integration/inventories/openvswitch.yaml:16]: lets delete this comment too, as it doesn't apply to ansible_connection: ssh

File: playbooks/network-ee-integration-tests/pre.yaml [hunk 3]
```diff
@@ -61,7 +61,7 @@
           appliance:
             vars:
               ansible_python_interpreter: /bin/python3.8
-      when: container_image_version in ["stable-2.9", "stable-2.11"]
+      when: container_image_version in ["stable-2.9", "stable-2.11"] and ansible_connection != "ssh"
```
Comment (pabelanger) [inline playbooks/network-ee-integration-tests/pre.yaml:64]: lets do this in a 2nd PR please

Comment (NilashishC) [reply playbooks/network-ee-integration-tests/pre.yaml:64]: Sure.

**Comment by pabelanger** [comment]
lets drop depends-on here for now

**Comment by pabelanger** [comment]
see changes in https://github.com/ansible/network-ee/pull/126 for the /usr/bin/python3.8

File: tests/integration/inventories/openvswitch.yaml [hunk 4]
```diff
@@ -0,0 +1,16 @@
...
+appliance:
+  hosts:
+    openvswitch:
+      ansible_host: openvswitch.example.org
+      ansible_user: zuul
+  vars:
+    ansible_connection: ssh
+    collection_name: openvswitch
+    collection_namespace: openvswitch
+    prepare_ovs_tests_task: False
```
Comment (pabelanger) [inline tests/integration/inventories/openvswitch.yaml:16]: lets remove this, I'd prefer not to leak testing configuration into the inventory file if possible.

If we want to do this, lets just update the collection to stop doing it.

**Comment by pabelanger** [comment]
once https://github.com/ansible-collections/vyos.vyos/pull/208 passes we can gate #125

**Comment by NilashishC** [comment]
@pabelanger I think we need to gate https://github.com/ansible/ansible-zuul-jobs/pull/1090 first?

**Comment by pabelanger** [comment]
yup, we'll go down the chain. And everything will auto enqueue

**Comment by pabelanger** [comment]
recheck

**Pull request merged**